.gitignore

```diff
@@ -3,6 +3,8 @@ yarn-debug.log
 yarn-error.log
 spicedb
 zed
+public/static/spicedb.wasm
+public/static/zed.wasm
```

Comment:
```suggestion
```

I'd gotten rid of these because I'm using jujutsu locally, which doesn't support git LFS. Gonna restore though.